src/App.js

```diff
@@ -1,7 +1,30 @@
 import "./App.css";
-
+import { Navbar } from "./components/Navbar/Navbar";
+import {
+  AddToCartPage,
+  LandingPage,
+  LoginPage,
+  ProductListing,
+  SignUpPage,
+  WishlistPage,
+} from "./pages/index";
+import Mockman from "mockman-js";
```

Comment:
you can move the library imports to the top.

Reply:
Yes will move it to top.